eng/common/scripts/get-markdown-files-from-changed-files.ps1

```diff
@@ -0,0 +1,49 @@
+param (
+  # The root repo we scaned with.
+  [string[]] $RootRepo = '$PSScriptRoot/../../..',
```

Comment:
I just noticed this but is there a reason this is a `string[]` as opposed to just a `string`?

Reply:
It should be string. Thanks for catching this.